packages/airnode-abi/package.json

```diff
@@ -6,12 +6,13 @@
   "main": "dist/index",
   "types": "dist/index",
   "files": [
-    "dist"
+    "dist",
+    "!dist/**/*.tsbuildinfo"
```

Comment:
You can use `npm publish --dry-run` to see what gets published. I checked every package and found two problems:
- `tsbuildinfo` files were published (addressed here)
- examples do not have `files` property set (addressed in https://github.com/api3dao/airnode/pull/1158/commits/1b504dc420fc406eed2a35957ce385bab8ac1bf0)